src/openms/source/ANALYSIS/DENOVO/CompNovoIdentificationBase.cpp

```diff
@@ -667,11 +667,11 @@ for (set<Size>::const_iterator it = used_pos.begin(); it != used_pos.end(); ++it
 
     for (set<ModificationDefinition>::const_iterator it = fixed_mods.begin(); it != fixed_mods.end(); ++it)
     {
-      ResidueModification mod = ModificationsDB::getInstance()->getModification(it->getModification());
+      ResidueModification mod = it->getModification();
       char aa = ' ';
       if (mod.getOrigin().size() != 1 || mod.getOrigin() == "X")
       {
-        cerr << "Warning: cannot handle modification " << it->getModification() << ", because aa is ambiguous (" << mod.getOrigin() << "), ignoring modification!" << endl;
+        cerr << "Warning: cannot handle modification " << mod.getName() << ", because aa is ambiguous (" << mod.getOrigin() << "), ignoring modification!" << endl;
```

Comment:
would globally replacing `cerr` with `LOG_WARN` make sense here?
This would allow attaching fancy streams in the background.